gprofiler/profilers/__init__.py

```diff
@@ -1,16 +1,17 @@
 # NOTE: Make sure to import any new process profilers to load it
-from gprofiler.profilers.dotnet import DotnetProfiler
-from gprofiler.profilers.java import JavaProfiler
+from gprofiler.platform import is_windows
 from gprofiler.profilers.perf import SystemProfiler
```

Comment:
What about the system profiler itself? It cannot be used in Windows either. AFAICT it won't be used if we pass `--no-perf` - we can make it the default in Windows (just like the default would be `--no-dotnet`, `--no-php` etc because these are not supported.

Reply:
Done. Moved SystemProfiler import to non-Windows group